### ISSUE TITLE
Parse city names 'San Francisco' wrong

### ISSUE THREAD

**Issue by ghost** [geopy/geopy#145]
I tried this code below:    

```
from geopy.geocoders import Nominatim
geolocator = Nominatim()
location = geolocator.geocode("San Francisco")
print location.address
```

And get result as：

> San Francisco, Agusan Del Sur, Caraga, Philippines

Obviously, it should be United States of America. What happened here? I am trying to use city name to find the country name.

**Comment by nps** [comment]
If you haven't done so already, update to 1.11.0, as this issue was resolved by [50e8da2](https://github.com/geopy/geopy/commit/50e8da299b31b4d3593e8ea0a76643958ac0fece#diff-b73b3356c6d20fb7bc201827f32cf963), which went out with this release. Compare:
```
>>> import geopy
>>> geopy.__version__
'1.10.0'
>>> from geopy.geocoders import Nominatim
>>> geolocator = Nominatim()
>>> location = geolocator.geocode("San Francisco")
>>> print location.address
San Francisco, Agusan Del Sur, Caraga, Philippines
```
```
>>> import geopy
>>> geopy.__version__
'1.11.0'
>>> from geopy.geocoders import Nominatim
>>> geolocator = Nominatim()
>>> location = geolocator.geocode("San Francisco")
>>> print location.address
SF, California, United States of America
```
NB: 'SF' is the result as returned by the call to Nominatim api.  It appears, for whatever reason, that a request of https://nominatim.openstreetmap.org/search?q=San+Francisco&format=json without the Accept-Language header (which is left unset by the base Geocoder & Nominatim) abbreviates the result for *just* San Francisco, California.  Compare:
```
curl "https://nominatim.openstreetmap.org/search?q=San+Francisco&format=json" 
```
with
```
curl -H "Accept-Language: en-US,en;q=0.5"  "https://nominatim.openstreetmap.org/search?q=San+Francisco&format=json"
```
but for whatever reason, adding an Accept-Language header to the geocoder (or doing so and changing the User-Agent) still return the abbreviated result.

**Comment by KostyaEsmukov** [comment]
Indeed, Nominatim responses depend on language. It also accepts a `country_bias` kwarg.
Compare:

```
>>> from geopy.geocoders import Nominatim
>>> geolocator = Nominatim()
>>> location = geolocator.geocode("San Francisco")
>>> print location.address
San Francisco, Agusan del Sur, Caraga, Philippines
```

```
>>> geolocator = Nominatim(country_bias='us')
>>> location = geolocator.geocode("San Francisco")
>>> print location.address
San Francisco, Brewster County, Texas, United States of America
```

```
>>> geolocator = Nominatim()
>>> location = geolocator.geocode("San Francisco", language="en")
>>> print location.address
San Francisco, San Francisco City and County, California, United States of America
```

It's a matter of documentation for a specific geocoding service (Nominatim), but not an issue with geopy. I'm closing this in favor of #268.
